package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "discord-interactions",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

Comment:
Even though it seems silly to bump to version 2.0.0 before the API is released, let's bump the major version because there's a backwards-incompatible change and some people are already using this library.